src/Combobox/index.js

```diff
@@ -363,11 +364,12 @@ const Combobox = ({
           value={inputValue}
           startIcon={icon}
           endContent={
+            inputEndContent === undefined ? (
```

Comment:
I was going to suggest a prop default, but I see the scope of the render is needed to decide the icon